lib/stats/DefaultStatsPrinterPlugin.js

```diff
@@ -57,12 +59,26 @@ const SIMPLE_PRINTERS = {
 			: undefined,
 	"compilation.time": (time, { bold }) => `Time: ${bold(time)}ms`,
 	"compilation.builtAt": (builtAt, { bold }) => {
+		if (!dateTimeFormat) {
+			dateTimeFormat = new Intl.DateTimeFormat([], {
```

Comment:
This should arguably be `'en'`, since the output is in English. Currently, English (hardcoded text) is mixed with whatever locale the environment is in.

Reply:
I wasn't sure if this was desired or not.  I guess 'time' above is English as well.  If it's a fixed locale especially, @mcollina ’s suggestion of using a different formatter especially makes sense.

Reply:
Makes sense, but in this case I don't even need DateTimeFormat and can save the initialization cost...